infra/base-images/base-builder/compile_python_fuzzer

```diff
@@ -24,6 +24,28 @@ fuzzer_package=${fuzzer_basename}.pkg
 PYFUZZ_WORKPATH=$SRC/pyfuzzworkdir/
 FUZZ_WORKPATH=$PYFUZZ_WORKPATH/$fuzzer_basename
 
+if [[ $SANITIZER = *introspector* ]]; then
+    # We must ensure python3.9, this is because we use certain
+    # AST logic from there.
+    # The below should probably be refined
+    apt install -y python3.9
+    apt-get install -y python3-pip
+    python3.9 -m pip install virtualenv
+    python3.9 -m virtualenv .venv
+    . .venv/bin/activate
+    pip3 install pyyaml
+    export PYTHONPATH="/fuzz-introspector/frontends/python/PyCG"
+
+    ARGS="--fuzzer $fuzzer_path"
+    if [ -n "${PYFUZZPACKAGE-}" ]; then
+      ARGS="$ARGS --package=${PYFUZZPACKAGE}"
+      ARGS="$ARGS --scan"
+    fi
+    python /fuzz-introspector/frontends/python/main.py $ARGS
```

Comment:
Does not go beyond this once `pycg` is install, here is the error:
```
INFO:__main__:Starting analysis
INFO:__main__:Running analysis with arguments: {fuzzer: /src/fuzz_glom.py, package: /src/glom/glom/ }
Traceback (most recent call last):
  File "/fuzz-introspector/frontends/python/main.py", line 354, in <module>
    sys.exit(main())
  File "/fuzz-introspector/frontends/python/main.py", line 342, in main
    exit_code = run_fuzz_pass(
  File "/fuzz-introspector/frontends/python/main.py", line 100, in run_fuzz_pass
    formatter = formats.Fuzz(cg)
AttributeError: module 'pycg.formats' has no attribute 'Fuzz'
```

Reply:
Checking `pycg` [repo](https://github.com/vitsalis/PyCG), there is no such `pycg.formats.Fuzz`.  Maybe it is your own developed module?

Reply:
Did you rebiuld `base-clang`? This is needed becaues you need the new version of fuzz-introspector that links to the custom PyCG -- see https://github.com/ossf/fuzz-introspector/tree/main/frontends/python

Reply:
Ah! that's the missing step, I just rebuilt `base-builder` and `base-builder-python`. Thanks for prompt response!